### PR TITLE
finishing up donut shop

### PR DESCRIPTION
**Pull request by hopeross**

### CODE REVIEW

File: Migrations/MyPlacesDbContext.cs [hunk 1]
```diff
@@ -49,7 +49,21 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         entity.Property(d => d.DonutShopName).IsRequired();
         entity.Property(d => d.DonutShopAddress);
         entity.Property(d => d.DonutShopWebsite);
-        entity.Property(d => d.DonutShopImage);
+        entity.Property(d => d.DonutShopCityState);
+        entity.HasData(
+            new { DonutShopId = 1, DonutShopName = "Five Daughters Bakery", DonutShopAddress = "1900 Eastland Ave Suite 101", DonutShopWebsite = "https://fivedaughtersbakery.com/", DonutShopCityState = "Nashville TN"},
+            new { DonutShopId = 2, DonutShopName = "Heavenly Donuts", DonutShopAddress = "1313 Shasta St", DonutShopWebsite = "https://www.heavenlydonut.com/", DonutShopCityState = "Redding CA"},
```
Comment (kendall-mchenry) [inline Migrations/MyPlacesDbContext.cs]: I know it's silly, but can we add a comma in the city/state for all of these? 
"Nashville, TN"

File: Models/Project/DonutShop.cs [hunk 2]
```diff
@@ -8,6 +8,6 @@ public class DonutShop
     public int DonutShopId { get; set; }
     public string DonutShopName { get; set; }
     public string DonutShopAddress { get; set; }
+    public string DonutShopCityState { get; set; }
     public string DonutShopWebsite { get; set; }
-    public string DonutShopImage { get; set; }
```
Comment (kendall-mchenry) [inline Models/Project/DonutShop.cs:12]: Did we decide to take the image out?

Comment (hopeross) [reply Models/Project/DonutShop.cs:12]: yes there was no easy way to get individual images

File: Migrations/MyPlacesDbContext.cs (hunk 1 above)
Comment (hopeross) [reply Migrations/MyPlacesDbContext.cs]: I guess?  Not sure why it matters?

Comment (kendall-mchenry) [reply Migrations/MyPlacesDbContext.cs]: I know it's a tiny detail, but that's proper grammar. :)

**Pull request merged**